### DISCUSSION TITLE
SketchUp SVG Importer 2024 Issue

### DISCUSSION THREAD

**Discussion by valerostudio** [community/community#120983] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Your SVG importer for SketchUp doesn't seem to be working in 2024. Is it possible to get an update to this extension?

Here is the error I get

Extension Errors Report
SketchUp: 24.0.484
OS: Windows 11 
Ruby: 3.2.2

Extension: SVG Import (0.6)
Error: RuntimeError (The `SortedSet` class has been extracted from the `set` library. You must use the `sorted_set` gem or other alternatives.)
C:/Program Files/SketchUp/SketchUp 2024/Tools/RubyStdLib/set/sorted_set.rb:4:in `rescue in <top (required)>'
C:/Program Files/SketchUp/SketchUp 2024/Tools/RubyStdLib/set/sorted_set.rb:1:in `<top (required)>'
<internal:C:/Program Files/SketchUp/SketchUp 2024/Tools/RubyStdLib/rubygems/core_ext/kernel_require.rb>:85:in `require'
<internal:C:/Program Files/SketchUp/SketchUp 2024/Tools/RubyStdLib/rubygems/core_ext/kernel_require.rb>:85:in `require'
C:/Users/mvalero/AppData/Roaming/SketchUp/SketchUp 2024/SketchUp/Plugins/svg_import/libraries/set.rb:425:in `<top (required)>'
<internal:C:/Program Files/SketchUp/SketchUp 2024/Tools/RubyStdLib/rubygems/core_ext/kernel_require.rb>:85:in `require'
<internal:C:/Program Files/SketchUp/SketchUp 2024/Tools/RubyStdLib/rubygems/core_ext/kernel_require.rb>:85:in `require'
C:/Users/mvalero/AppData/Roaming/SketchUp/SketchUp 2024/SketchUp/Plugins/svg_import/loader.rb:49:in `<module:SVGImport>'
C:/Users/mvalero/AppData/Roaming/SketchUp/SketchUp 2024/SketchUp/Plugins/svg_import/loader.rb:37:in `<top (required)>'
C:/Program Files/SketchUp/SketchUp 2024/Tools/extensions.rb:197:in `require'
C:/Program Files/SketchUp/SketchUp 2024/Tools/extensions.rb:197:in `load'
C:/Users/mvalero/AppData/Roaming/SketchUp/SketchUp 2024/SketchUp/Plugins/svg_import.rb:57:in `register_extension'
C:/Users/mvalero/AppData/Roaming/SketchUp/SketchUp 2024/SketchUp/Plugins/svg_import.rb:57:in `<module:SVGImport>'
C:/Users/mvalero/AppData/Roaming/SketchUp/SketchUp 2024/SketchUp/Plugins/svg_import.rb:34:in `<top (required)>'

**Comment by ebndev** [comment]
Hi @valerostudio,

Thanks for being a part of the GitHub Community, we're glad you're here!

The Profile category is for conversations related to your GitHub profile page. I have moved this post to General, however, it looks like you're looking for support from Sketchup. You might want to try asking for help somewhere that focuses on this project, such as opening an issue in the [project's repository](https://github.com/SketchUp) or visiting [their website](https://forums.sketchup.com/c/developers/12).

It's possible that another GitHub user might have run into this same issue and can help, but we want to make sure you’re getting the best support you can and that your message goes where you intended.

Best of luck!

**Comment by ebndev** [comment]
Hi again @valerostudio!

If you don't mind, would you mind letting us know how you initially ended up in the [Profile category](https://github.com/orgs/community/discussions/categories/profile?discussions_q=is%3Aopen+category%3AProfile+) when creating your discussion? We are trying to investigate if there is a bug or link that may be re-directing users from the project's Discussions board, if that’s where you intended to post.

Any insight would be helpful, thanks!

**Comment by valerostudio** [comment]
Honestly, I have no idea how I did it. I was trying to reach out to the
developer and couldn't find a way to do so.

On Mon, Jun 17, 2024 at 6:50 PM ebndev ***@***.***> wrote:

> Hi again @valerostudio <https://github.com/valerostudio>!
>
> If you don't mind, would you mind letting us know how you initially ended
> up in the Profile category
> <https://github.com/orgs/community/discussions/categories/profile?discussions_q=is%3Aopen+category%3AProfile+>
> when creating your discussion? We are trying to investigate if there is a
> bug or link that may be re-directing users from the project's Discussions
> board, if that’s where you intended to post.
>
> Any insight would be helpful, thanks!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/120983#discussioncomment-9799621>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BCTIGUCC36VP5X7IBLXWGMDZH5R45AVCNFSM6AAAAABGW2FUTCVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TOOJZGYZDC>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
*Matthew Valero*
Visualization Artist
* Valero Studio *

**Reply by ebndev** [reply]
That's a very valid response lol. Thanks for taking the time to answer anyway!
